miio/devicestatus.py

```diff
@@ -2,7 +2,16 @@
 import logging
 import warnings
 from enum import Enum
-from typing import Callable, Dict, Optional, Union, get_args, get_origin, get_type_hints
+from typing import (
```

Comment:
Do you mind extracting the devicestatus changes into a separate PR? This will make using `git annotate` nicer in case someone in the future wonders why the API got changed.

Reply:
https://github.com/rytilahti/python-miio/pull/1519